src/cascade/core/context.py

```diff
@@ -36,51 +37,17 @@ def __iter__(self):
         return iter([self.pini, self.iota, self.rho, self.chi, self.omega])
 
 
-class _Outputs:
-    __slots__ = ["integrands"]
-
-    def __init__(self):
-        self.integrands = _Integrands()
-
-
-class _Integrand:
-    __slots__ = ["name", "age_ranges", "time_ranges", "value_covariate_multipliers", "std_covariate_multipliers"]
+class _IntegrandCovariateMultiplier:
+    __slots__ = ["name", "value_covariate_multipliers", "std_covariate_multipliers"]
```

Comment:
Darn it. There are 3 types of covariate multipliers, not two. You can have one on the value of an integrand, the stdev of an integrand OR on the value of a rate. So if you put a covariate multiplier on "remission" value, that's different from a covariate multiplier on "iota" value. I don't think this setup expresses the rate covariate multipliers. The avgint table can produce not only integrands, which is what the name implies, but rates, too.